evo/tools/plot.py

```diff
@@ -383,7 +385,7 @@ def traj_colormap(ax: plt.Axes, traj: trajectory.PosePath3D,
     ])
     if title:
         ax.legend(frameon=True)
-        plt.title(title)
+        ax.set_title("title")
```

Comment:
you replaced the variable with a string literal, fix it please

Reply:
I am sorry, i dont know how i missed that